client/src/components/Thread.js

```diff
@@ -20,12 +21,13 @@ class Thread extends React.Component {
                       <h3>Respond to this post</h3>
                       <form id="thread-form">
```

Comment:
This form appears twice on the page. Please delete lines 22-28 and leave the CreateComment tag, which already has a form.